README.md

```diff
@@ -2,14 +2,35 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/Mellanox/network-operator)](https://goreportcard.com/report/github.com/Mellanox/network-operator)
 [![Build Status](https://travis-ci.com/Mellanox/network-operator.svg?branch=master)](https://travis-ci.com/Mellanox/network-operator)
 
-# [WIP] Nvidia Mellanox Network Operator
-
+- [Nvidia Mellanox Network Operator](#nvidia-mellanox-network-operator)
+  * [Prerequisites](#prerequisites)
+    + [Kubernetes Node Feature Discovery (NFD)](#kubernetes-node-feature-discovery--nfd-)
+  * [Resource Definitions](#resource-definitions)
+    + [NICClusterPolicy CRD](#nicclusterpolicy-crd)
+      - [NICClusterPolicy spec:](#nicclusterpolicy-spec-)
+        * [Example for NICClusterPolicy resource:](#example-for-nicclusterpolicy-resource-)
+      - [NICClusterPolicy status:](#nicclusterpolicy-status-)
+        * [Example of the `Status` field of a NICClusterPolicy instance:](#example-of-the--status--field-of-a-nicclusterpolicy-instance-)
+  * [System Requirements](#system-requirements)
+  * [Deployment Example](#deployment-example)
+  * [Driver Containers](#driver-containers)
+  * [Future work](#future-work)
+
+<small><i><a href='http://ecotrust-canada.github.io/markdown-toc/'>Table of contents generated with markdown-toc</a></i></small>
+
+# Nvidia Mellanox Network Operator
 Nvidia Mellanox Network Operator leverages [Kubernetes CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
 and [Operator SDK](https://github.com/operator-framework/operator-sdk) to manage Networking related Components in order to enable Fast networking, 
 RDMA and GPUDirect for workloads in a Kubernetes cluster.
 
-## Prerequisites
+The Goal of Network Operator is to manage _all_ networking related bits a pieces to enable execution of
```

Comment:
related bits a pieces?

Reply:
bits _and_ pieces ?
or do you prefer the more formal : "related components"